types/types.go

```diff
@@ -165,3 +165,8 @@ func GetDiskConditionFromStatus(status DiskStatus, conditionType DiskConditionTy
 	}
 	return condition
 }
+
+type DiskUpdateInfo struct {
+	DiskSpec
+	StorageMaximum int64 `json:"storageMaximum"`
```

Comment:
I think we should able to set `ReservedSpace` instead of maximum directly for the testing? What do you think?

Reply:
Yeah, that sounds nice and more like user operation. I think it maybe more difficult to test edge situation of scheduler. We need to recalculate reserved space for disks in different cluster.

Reply:
I see. But it sounds weird to have the option in the status but an API able to update it (and user suppose not to update it by themselves). I think it makes sense to do a bit extra work on the calculation.

Reply:
What about remove update maximum field from API in this release? I will dig more in longhorn-test later.